app/uk/gov/hmrc/tradergoodsprofiles/controllers/actions/ValidateHeaderAction.scala

```diff
@@ -71,9 +71,8 @@ class ValidateHeaderAction @Inject() (uuidService: UuidService)(implicit ec: Exe
 
   private def validateContentTypeHeader(request: Request[_]): Future[Boolean] =
     request.method match {
-      case "DELETE" =>
-        successful(true)
-      case _        =>
+      case "DELETE" | "GET" => successful(true)
```

Comment:
Here we may want to do something like the ValidationRules in the router. This will allow us to call the right action for the  right method. By that I mean we now which action to call for which method so we can abide to do these case statements.

Give me a shout if you want any help.

Reply:
I have refactored the `ValidateHeaderAction` to use method-specific `Content-Type` validation
- Created a ValidationActions trait with `DefaultValidation` and `NoContentTypeValidation`
- Mapped the HTTP methods to corresponding validations to  determine the content-type validation logic